relayer-cli/src/commands/misbehaviour.rs

```diff
@@ -0,0 +1,129 @@
+use abscissa_core::{config, error::BoxError, Command, Options, Runnable};
+use ibc::events::IbcEvent;
+use ibc::ics02_client::events::UpdateClient;
+use ibc::ics02_client::height::Height;
+use ibc::ics24_host::identifier::{ChainId, ClientId};
+use ibc_relayer::chain::handle::ChainHandle;
+use ibc_relayer::foreign_client::ForeignClient;
+
+use crate::application::CliApp;
+use crate::cli_utils::spawn_chain_runtime;
+use crate::conclude::Output;
+use crate::prelude::*;
+use ibc::ics02_client::client_state::ClientState;
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct MisbehaviourCmd {
+    #[options(
+        free,
+        required,
+        help = "identifier of the chain where client updates are monitored for misbehaviour"
+    )]
+    chain_id: ChainId,
+
+    #[options(
+        free,
+        required,
+        help = "identifier of the client to be monitored for misbehaviour"
+    )]
+    client_id: ClientId,
+}
+
+impl Runnable for MisbehaviourCmd {
+    fn run(&self) {
+        let config = app_config();
+
+        let res = monitor_misbehaviour(&self.chain_id, &self.client_id, &config);
+        match res {
+            Ok(()) => Output::success(()).exit(),
+            Err(e) => Output::error(format!("{}", e)).exit(),
+        }
+    }
+}
+
+pub fn monitor_misbehaviour(
+    chain_id: &ChainId,
+    client_id: &ClientId,
+    config: &config::Reader<CliApp>,
+) -> Result<(), BoxError> {
+    let chain = spawn_chain_runtime(&config, chain_id)
```

Comment:
Note for follow-up: At some point later down the line we could use the `Registry` to avoid spawning a runtime multiple time (eg. if there are more than one client per chain).

Reply:
Agreed. I was just about to add it but we decided to only monitor one client with the CLI for now.